flujo/application/runner.py

```
@@ -909,3 +931,15 @@ async def _shutdown_state_backend(self) -> None:
     "_accepts_param",
     "_extract_missing_fields",
 ]
+
+# Register the default runner factory to break circular dependencies with Pipeline
+from ..domain.interfaces import set_default_runner_factory as _set_default_runner_factory  # noqa: E402
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)